src/specVersions/v0.2/RicardianContractProcessorImpl.test.ts

```diff
@@ -0,0 +1,88 @@
+import { Abi } from '../../interfaces'
+import { RicardianContractProcessorImpl } from './RicardianContractProcessorImpl'
+
+import { eosioTokenAbi, transferTransaction } from '../../testfixtures/fixtures'
+
+const transaction = JSON.parse(transferTransaction)
+const abi = JSON.parse(eosioTokenAbi)
+
+// tslint:disable:max-line-length
```

Comment:
is this tslint disable needed here?

Reply:
That's actually turning it off for the whole file. Could switch back to having a bunch of `disable-next-line` comments, tho there were so many in weird places it's why I went to just killing it for the whole file.

Reply:
Gotcha, assumed it was per line, since i saw `// tslint:disable:max-line-length` repeated a few times throughout this file

Reply:
Those were some extraneous ones that I forgot to clean out. I think I clean all that up now.